frontend/ui/widget/virtualkeyboard.lua

```diff
@@ -130,6 +130,7 @@ function VirtualKey:init()
         self.hold_callback = function()
             self.ignore_key_release = true
             self.keyboard:onHideKeyboard()
+            self:update_keyboard(false, true)
```

Comment:
I'm curious where you experience the need for this?

This doesn't have anything to do with borders (which, border, actually?): it just repaints a single *key*, something which happened right before already?

Reply:
It is needed in Text editor, hiding the keyboard shouldn't work there because of nav_bar, the arrow down border remains black after hold.

Reply:
I'll check that tomorrow, but, in that case, move it *before* the onHideKeyboard please ;).

(Or, better yet, only do it when necessary ^^).

Reply:
Moved